.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Please remove the `.github/ISSUE_TEMPLATE` directory. This is no longer needed now that we have a [`aws-controllers-k8s/.github` repository](https://github.com/aws-controllers-k8s/.github/). We do still need `.github/workflows`, though.